packages/donate-button-v4/src/public/index.html

```diff
@@ -12,12 +12,6 @@
 		<div id="edo-donate-btn">
 			<a href="https://www.every.org/closegap#donate">Donate</a>
 		</div>
-		<script
```

Comment:
@Fishbakh-N I think these lines are required for production (but not for localhost)

Reply:
[index.html](https://github.com/everydotorg/donate-button/pull/332/files#diff-e4dfd4c1ffc1fc679c4ef2c2ef315296502a77550cab6836adc8a265d109d6d6) file is needed only for demo. and script attaches on build step.